includes/payments/class-edd-payment.php

```diff
@@ -1898,6 +1898,10 @@ public function refund() {
 	 */
 	public function update_status( $status = '' ) {
 
+		if ( ! $this->order ) {
+			return;
```

Comment:
```suggestion
			return false;
```

Looks like this method is meant to return a boolean.